### PR TITLE
Component collection

### PR DESCRIPTION
**Pull request by mdrisco4**
May or may not need a bit of fine tuning, but have it pretty close to mockup.  I never use styled components and am new to using a Theme so much of the day was spent getting familiar with both of those and applying it here.  Moving forward I should be able to move much quicker with my workload now that I am familiar with everything being utilized with styling.

### CODE REVIEW

File: src/components/component-collection/ComponentCollection.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  background-color: ${props => props.theme.colors.primaryPink};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    height: 280px;
+    padding: 30px 10px 10px 10px;
+  }
+`;
+
+const GownImageMobile = styled.img`
+  width: 160px;
+  height: 150px;
```
Comment (erinkelley27) [inline src/components/component-collection/ComponentCollection.js]: Try taking the height and width directly from the figma for this one and the desktop version

File: src/components/component-collection/ComponentCollection.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    margin: 8px 15px 12px 15px;
+    width: 160px;
+    height: 180px;
+  }
+`;
+
+const Title = styled.p`
+  text-align: center;
+  font-weight: bold;
+  font-family: ${props => props.theme.fontStyles.h1Styles.fontStyle};
```
Comment (erinkelley27) [inline src/components/component-collection/ComponentCollection.js]: For the Desktop version, it looks like the font is Raleway.

File: src/components/component-collection/ComponentCollection.stories.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import ComponentCollection from "./ComponentCollection";
+
+import imageFile1 from "./images/test-image-1.png";
+import imageFile2 from "./images/test-image-2.png";
+import imageFile3 from "./images/test-image-3.png";
+import imageFile4 from "./images/test-image-4.png";
```
Comment (erinkelley27) [inline src/components/component-collection/ComponentCollection.stories.js]: I would move these images to the `/assets` directory and update the path. Also, give your images semantic names, so their names describe what they are. Even something like `fall-collection-1.png`, etc. would be good. There are a lot of developers that will be testing images in this build, so we want to make sure they don't have to look through the images to figure out what they are.

**Comment by mdrisco4** [comment]
I  think it should be good to go now.

**Comment by erinkelley27** [comment]
@mdrisco4 Looks like there are some merge conflicts here - let me know when you fix them and I'll review.

**Comment by mdrisco4** [comment]
Fixed conflicts.  Should be good to go now.

**Comment by erinkelley27** [comment]
@mdrisco4 I am getting an error that `variables` is declared twice in `.storybook/config.js`. Can you remove on variable declaration, add commit and push?

**Comment by mdrisco4** [comment]
good to go

**Pull request merged**